misc/python/materialize/checks/identifiers.py

```diff
@@ -272,9 +272,9 @@ def validate(self) -> Testdrive:
         {dq_print(self.ident["schema"])}
 
         > SHOW SINKS FROM {dq(self.ident["schema"])};
-        {dq_print(self.ident["sink0"])} kafka ${{arg.default-storage-size}}
-        {dq_print(self.ident["sink1"])} kafka ${{arg.default-storage-size}}
-        {dq_print(self.ident["sink2"])} kafka ${{arg.default-storage-size}}
+        {dq_print(self.ident["sink0"])} kafka ${{arg.default-storage-size}} {dq(self.ident["db"])}_{dq(self.ident["schema"])}_{dq(self.ident["sink0"])}
+        {dq_print(self.ident["sink1"])} kafka ${{arg.default-storage-size}} {dq(self.ident["db"])}_{dq(self.ident["schema"])}_{dq(self.ident["sink1"])}
+        {dq_print(self.ident["sink2"])} kafka ${{arg.default-storage-size}} {dq(self.ident["db"])}_{dq(self.ident["schema"])}_{dq(self.ident["sink2"])}
```

Comment:
For some reason it looks like in the test output there's spaces inbetween the db, schema, and idents:
```
Poor diff:
- 15.0 kafka 4-1 -1.0 _0.0 _15.0
+ 15.0 kafka 4-1 -1.0_0.0_15.0
- 16.0 kafka 4-1 -1.0 _0.0 _16.0
+ 16.0 kafka 4-1 -1.0_0.0_16.0
- 17.0 kafka 4-1 -1.0 _0.0 _17.0
+ 17.0 kafka 4-1 -1.0_0.0_17.0
```